src/routes/apiKeys.ts

```
@@ -15,15 +16,25 @@ import {
   HttpStatusCode,
 } from "../utils/errorHandler.js";
 import { prisma } from "../lib/prisma.js";
+import { validateScopes, ALL_SCOPES } from "../utils/permissions.js";
 
 export default async function apiKeysRoute(fastify: FastifyInstance) {
   // POST /api-keys - Create a new API key
   fastify.post("/api-keys", {
     schema: {
+      definitions: {
+        CreateApiKeyRequestBody: Type.Object({
```

Biome (2.1.2)

[error] This parameter is used before its declaration.

The parameter is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)